public/app/features/dashboard-scene/scene/VariableControls.tsx

```diff
@@ -15,18 +15,45 @@ import {
 import { useElementSelection, useStyles2 } from '@grafana/ui';
 
 import { DashboardScene } from './DashboardScene';
+import { DrilldownControls } from './DrilldownControls';
 import { AddVariableButton } from './VariableControlsAddButton';
 
 export function VariableControls({ dashboard }: { dashboard: DashboardScene }) {
   const { variables } = sceneGraph.getVariables(dashboard)!.useState();
+  const styles = useStyles2(getStyles);
+
+  // Get visible variables for drilldown layout
+  const visibleVariables = variables.filter((v) => v.state.hide !== VariableHide.hideVariable);
```

Comment:
I just realized that no panels will run their queries because of this line here -- the scopes Variable is a hidden one, so you would be losing that too later in `restVariables`, which would then miss setting up the scopes var and thus keep it forever in loading which won't run the queries in the SQR later down the line. 

So I reckon we only need to filter for vars that are not `VariableHide.inControlsMenu`, and leave the `VariableHide.hideVariable` there